indico/modules/users/templates/_favorites.html

```diff
@@ -1,9 +1,16 @@
 {% macro favorite_users_list(user) %}
     {% for fav in user.favorite_users|sort(attribute='full_name') %}
-        <li data-user-id="{{ fav.id }}" class="js-user-row">
+        <li data-user-id="{{ fav.id }}" class="flexrow f-a-center js-user-row">
             <span>{{ fav.full_name }}</span>
-            <a class="toggle icon-close right js-delete-user" title="{% trans %}Remove from favourites{% endtrans %}" href="#"
-               data-href="{{ url_for('.user_favorites_user_remove', fav_user_id=fav.id) }}"></a>
+            <span class="right">
+                <span class="ui icon tiny button right js-delete-user"
+                     data-tooltip="{% trans %}Remove from favourites{% endtrans %}"
+                     data-inverted=""
```

Comment:
Is this attribute handled somehwere? (ahh I guess it is handled by SUIR)

Reply:
I asked him the same on mattermost since I also didn't find anything: It's using the pure-css tooltips from SUI.

Reply:
Ah, ok. I thought so..

Reply:
I ended up removing this because it was starting to look very odd, having different tooltips. When I came to myself I was already changing the tooltips in five files. So I rolled it back.